andromeda/app.py

```diff
@@ -0,0 +1,14 @@
+# Adds logic to serve static html for running within huggingface
+from flask import send_from_directory, request
+from main import app
+
+@app.errorhandler(404)
+def page_not_found(e):
+    path = request.path.lstrip('/')
+    if path == '':
+       path = 'index.html'
+    print(path)
```

Comment:
```suggestion

```
See @johnbradley's [comment/suggestion](https://github.com/Imageomics/Andromeda/pull/74#discussion_r1370143678) on this.